src/Matcher_lexer.mll

```diff
@@ -29,7 +29,7 @@ let white = [' ' '\t']+
 let alpha = ['a' - 'z' 'A' - 'Z']
 let modifier = alpha+ ['-' '+']
 
-let binding = ['a'-'z' 'A'-'Z' '0'-'9' '`' '-' '=' '[' ']' '\\' ';' '\'' ',' '.' '/']
+let binding = ['a'-'z' 'A'-'Z' '0'-'9' '`' '-' '=' '[' ']' '\\' ';' '\'' ',' '.' '/' '(' ')' '|' '_' '%' '+']
```

Comment:
Next question: Why is this inclusive instead of exclusive? Could it not either just exclude characters with "special" meaning, or match any single character as the last case of the production, after any cases that matches "special" characters?